alkira/resource_alkira_connector_azure_expressroute.go

```diff
@@ -153,6 +153,90 @@ func resourceAlkiraConnectorAzureExpressRoute() *schema.Resource {
 							Optional: true,
 							Elem:     &schema.Schema{Type: schema.TypeInt},
 						},
+						"segment_options": {
+							Description: "Instance level segment specific routing and gateway configurations." +
+								"Only required when `tunnel_protocol` is `IPSEC`.",
+							Type:     schema.TypeList,
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"segment_name": {
+										Description: "The name of an existing segment in the Alkira environment.",
+										Type:        schema.TypeString,
+										Required:    true,
+									},
+									"customer_gateways": {
+										Description: "Customer gateway configurations for `IPSEC` tunnels. " +
+											"Required only if `tunnel_protocol` is `IPSEC`.",
```

Comment:
The block is already required only when tunnel_protocol is IPSEC, so there is no point to claim this again in the field.